src/main/java/bio/terra/service/auth/iam/IamAction.java

```diff
@@ -10,9 +10,13 @@ public enum IamAction {
   // common
   CREATE,
   DELETE,
+  GET,
+  UPDATE,
+  APPROVE,
   READ_POLICY,
   READ_POLICIES,
   SHARE_POLICY_READER("share_policy::reader"),
+  SHARE_POLICY_AGGREGATE_DATA_READER("share_policy::aggregate_data_reader"),
```

Comment:
Do we also need the share policy for the SNAPSHOT_BUILDER_MANAGER  role?

Reply:
No, because for the time being that will only be shared by admins, who have the `ALTER_POLICIES` action.